src/components/viewer.ts

```diff
@@ -180,22 +143,7 @@ export class Viewer {
     openExternal(sourceFile: string) {
         const pdfFile = this.extension.manager.tex2pdf(sourceFile)
         const configuration = vscode.workspace.getConfiguration('latex-workshop')
-        let command = JSON.parse(JSON.stringify(configuration.get('view.pdf.external.command'))) as ExternalCommand
```

Comment:
This should probably not be removed. It was added in commit d9bb923506a987021b47bb45cde2d27f6f7bbed4 in order to have a good default value on all platforms because specifying a platform specific default configuration isn't supported by vscode.

Reply:
I confirm this should be kept along with the following `switch (process.platform)` to set a default value for the external viewer depending on the os.

Reply:
Never checked that. I copied the code from a too old commit.